excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/parser/DisplayGroupParserTest.java

```diff
@@ -401,14 +405,14 @@ void shouldNotParseCaseTypeTabForInvalidUserRoles() throws InvalidShowConditionE
         MapperException thrown = assertThrows(MapperException.class,
             () -> caseTypeTabParser.parseAll(definitionSheets));
         assertThat(thrown.getMessage(),
-            is("- Invalid idam or case role 'Role1' in 'CaseTypeTab' tab for TabId 'NameTab'"));
+            is("- Invalid IDAM access profile or case role 'AccessProfile1' in 'CaseTypeTab' tab "
```

Comment:
Rename to remove refernece to IDAM.

Reply:
Fixed in https://github.com/hmcts/ccd-definition-store-api/pull/961/commits/3951e2bdda580027372daf452a54fbcde899194f